tests/conftest.py

```diff
@@ -11,7 +11,12 @@ def pytest_addoption(parser):
 
 def pytest_configure(config):
     config.addinivalue_line("markers", "slow: mark test as slow to run")
+    import sys
+    sys._called_from_test = True  # currently used to set logging level
 
+def pytest_unconfigure(config):
+    import sys
+    del sys._called_from_test
```

Comment:
What is this `_called_from_test` property? Are we creating a new property on the built-in `sys` module? This is not very idiomatic; what is the intent of this design decision? Are we trying to adjust logging output only during test execution? That should be done from the `logging` package configuration --- not by editing bulitin modules

Reply:
Agreed.